nima/webserver/webserver/src/main/java/io/helidon/nima/webserver/http/Filters.java

```diff
@@ -111,7 +116,7 @@ public void proceed() {
                 return;
             }
             if (filters.hasNext()) {
-                errorHandlers.runWithErrorHandling(ctx, request, response, this::runNextFilter);
+                filters.next().filter(this, request, response);
```

Comment:
The problem I see with this new approach is that if an error handler is executed for routing, it would get executed again for the whole `executeFilters` once again.

I think that if the same exception that was thrown from routing would enter the second error handler (for filtering), the error handler should not be executed (same instance of exception).